src/app/content/highlights/components/Confirmation.spec.tsx

```diff
@@ -14,28 +20,54 @@ jest.mock('react', () => {
 });
 
 describe('Confirmation', () => {
+  const cache = createIntlCache();
+  let store: Store;
+  let services: ReturnType<typeof createTestServices>;
+
+  const intl = createIntl({
+    locale: 'en',
+    messages: {
+      ...en,
+      'i18n:confirm': 'confirm',
+      'i18n:message': 'message',
+    },
+  }, cache);
```

Comment:
@TomWoodward this was the offending file re "missing message" errors. Is this the fix you had in mind?

Reply:
sort of, although https://formatjs.io/docs/intl#definemessagesdefinemessage is also a thing that i was imagining using instead of replacing/duplicating the initialization

Reply:
I don't understand how that function helps—wouldn't I still need to pass the new messages to a provider so `<FormattedMessage>` can locate the ID?

Reply:
@TomWoodward don't think this is a blocker for the PR but I'm still not clear on how to implement this in jest

Reply:
i thought defineMessage could be called on `intl` to add more messages, it doesn't work like that i was wrong. but couldn't you overwrite the intl in the test services with the one you're creating and still use `TestContainer` by passing services in?